Files/Filesystem/ListedItem.cs

```diff
@@ -292,7 +290,7 @@ public override string ToString()
         public bool IsShortcutItem => this is ShortcutItem;
         public bool IsLinkItem => IsShortcutItem && ((ShortcutItem)this).IsUrl;
 
-        public bool IsPinned => App.SidebarPinnedController.Model.Items.Contains(itemPath);
+        public bool IsPinned => App.SidebarPinnedController.Model.LibraryItems.Contains(itemPath);
```

Comment:
What's the goal of this change?

Reply:
Yes, i forget to change LibraryItems to FavoriteItems